homeTask1.html

```diff
@@ -0,0 +1,333 @@
+            padding: 5px;
+            border: solid black 2px;
+        }
+
+        #allNews {
+            align-self: flex-end;
+        }
+
+        .post-text {
+            position: relative;
+            overflow: hidden;
+                display: -webkit-box;
+                -webkit-line-clamp: 6;
+                -webkit-box-orient: vertical;
+        }
+
+        .main-heading {
+            flex-direction: row;
+        }
+      
+        /* for screen sizes less than 600px, usually phones and tabs */
+        @media only screen and (max-width: 600px) {
+            .post-content {
+                width: 50%;
+            }
+            
+            /* add ellipsis after 3 lines in article detail content */
+            .post-text {
+                overflow: hidden;
+                display: -webkit-box;
+                -webkit-line-clamp: 3;
+                -webkit-box-orient: vertical;
+            }
+
+            .main-heading {
+                flex-direction: column;
+            }
+            /* aligning All News to start of the flex */
+            #allNews {
+                align-self: flex-start;
+                padding-bottom: 10px;
+            }
+
+            .subscribe {
+                margin-left: -80px;
+                position: fixed;
+            }
+            
+        }
+        @media only screen and (max-width: 992px){
+            .post-content {
+                width: 70%;
+            }
+            
+            /* add ellipsis after 3 lines in article detail content */
+            .post-text {
+                overflow: hidden;
+                display: -webkit-box;
+                -webkit-line-clamp: 5;
+                -webkit-box-orient: vertical;
+            }
+
+            /* .main-heading {
+                flex-direction: row;
+            } */
+            /* aligning All News to start of the flex */
+            #allNews {
+                align-self: center;
+                padding-bottom: 10px;
+            }
+
+            .subscribe {
+                
+                font-size: 16px;
+            }
+            .input {
+            width: 11em;
+            font-size: 16px;
+            padding: 5px;
+        }
+        #allNews {
+                align-self: center;
+                
+            }
+            .column{
+                align-items: flex-end;
+            }
+            
+        }
+       
+
+    </style>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
+    <header class="header container">
+        <div class="header-box">
+            <h1 class="headings">NEWS BLOG</h1>
+            <h4 class="headings">WORLDWIDE NEWS</h4>
+        </div>
+
+    </header>
+    <main class="container">
+        <div class="flex-container main-heading">
+            <div >
+                <span>SELECT CATEGORY</span>
+                <br>
```

Comment:
don't use br here, instead convert the span to div, if u want it to take full space